static/less/search.less

```diff
@@ -221,10 +221,10 @@
 
 // Search results page list:
 .search-results-header {
-    color: #000;
+    color: @text-color-highlight;
     font-size: 12px;
     line-height: 1.5;
     padding: 10px 0 15px;
     margin-bottom: 10px;
-    border-bottom: 1px solid #ddd;
```

Comment:
@felixhahnweilheim thank you for looking into this! Lucas and I have agreed, that, instead of using functions like darken and whiten, we should swap the color to the closest looking variable. In this case, it would change from `#ddd` to the variable `@backrgound3`

<img width="1034" alt="Screenshot 2024-05-22 at 4 43 05 PM" src="https://github.com/humhub/humhub/assets/105652258/a6763e7d-8bd5-4fb4-a702-6c36e48ecc6a">

This would mean, that the color `#ddd` would always have to be changed to `@background3`. We also use this color here, i.eg: `#wall-stream-filter-nav .wall-stream-filter-head` 

Please also find out, whether we use the color somewhere I didn't mention yet.